go/libraries/doltcore/row/tagged_values.go

```diff
@@ -277,6 +277,8 @@ func CountCellDiffs(from, to types.Tuple) (uint64, error) {
 
 	for i, v := range f {
 		ov, ok := t[i]
+		// TODO : if the to has nil value, it can be drop column, so should not be cell change?
```

Comment:
Interesting! If this method had the to/from schemas passed in, would that be enough to differentiate between these two cases?